src/Mod/Draft/draftguitools/gui_base.py

```diff
@@ -32,6 +32,8 @@
 import draftutils.todo as todo
 from draftutils.messages import _msg, _log
 
+__metaclass__ = type  # to support Python 2 use of `super()`
+
 
 class GuiCommandSimplest:
```

Comment:
@vocx-fc , wasn't it simpler to use` class GuiCommandSimplest(object):`?

Reply:
New style classes don't need to specify `object` at all in Python 3. So, by adding `__metaclass__` we keep Python 2 compatibility, and it's just one line that later we can remove without touching the classes.